src/main/java/io/github/janmalch/kino/control/reservation/GetMyReservationsControl.java

```diff
@@ -27,12 +26,7 @@ public GetMyReservationsControl(String myAccountName) {
     var reservationDtos =
         reservations
             .stream()
-            .map(
-                r -> {
-                  var success =
-                      new GetReservationByIdControl(r.getId()).execute(new EitherResultBuilder<>());
-                  return success.getSuccess().getData();
-                })
+            .map(r -> new GetReservationMapper().map(r, ReservationInfoDto.class))
```

Comment:
Muss hier für jede gefundene Reservation ein neuer Mapper erstellt werden?

Reply:
Mapper Instanziierung jetzt außerhalb des Streams

Reply:
creating instance of mapper now outside of stream

https://github.com/JanMalch/kino/blob/15926a15881e5d23ff95161a94a56610538cc51b/src/main/java/io/github/janmalch/kino/control/reservation/GetMyReservationsControl.java#L23